pkg/services/dashboards/accesscontrol_test.go

```diff
@@ -27,7 +27,7 @@ func TestNewFolderNameScopeResolver(t *testing.T) {
 		orgId := rand.Int63()
 		title := "Very complex :title with: and /" + util.GenerateShortUID()
 		// nolint:staticcheck
```

Comment:
We should be careful about leftover nolint directives - they would become hard to notice and to remove later

Reply:
not really I am searching for this lines in pkgs, that's my method for removing them, but will remove this one